docker-compose/monitor/Makefile

```diff
@@ -0,0 +1,50 @@
+.PHONY: build
+build: clean-jaeger
+	cd ../../ && \
+	make build-all-in-one && \
+	make docker-images-jaeger-backend && \
+	docker tag jaegertracing/all-in-one:latest jaegertracing/all-in-one:dev
+
+# run starts up the system required for SPM using the latest jaeger and otel images.
+.PHONY: run
+run: export JAEGER_IMAGE_TAG = latest
+run: _run-connector
+
+# run starts up the system required for SPM using the latest otel image and a development jaeger image.
+# Note: the jaeger "dev" image can be built with "make build".
+.PHONY: run-dev
+run-dev: export JAEGER_IMAGE_TAG = dev
+run-dev: _run-connector
+
+# _run-connector is the base target to bring up the system required for SPM using the new OTEL spanmetrics connector.
+.PHONY: _run-connector
+_run-connector: export OTEL_IMAGE_TAG = latest
+_run-connector: export OTEL_CONFIG_SRC = ./otel-collector-config-connector.yml
+_run-connector: export PROMETHEUS_QUERY_SUPPORT_SPANMETRICS_CONNECTOR = true
+_run-connector: export PROMETHEUS_QUERY_NAMESPACE = span_metrics
+_run-connector: export PROMETHEUS_QUERY_DURATION_UNIT = ms
```

Comment:
when running collector with connector, does it require these args?  If not, can we not make these the default values and only require overriding them when different?

Reply:
Nope, it doesn't. I just put it there for illustrative purposes. Removed in: [17f42b7](https://github.com/jaegertracing/jaeger/pull/4452/commits/17f42b70209bb85f8c78627295a4838e94d8adb0).